### ISSUE TITLE
Outdated version number

### ISSUE THREAD

**Issue by NathanBnm** [matfantinel/reminduck#26]
Since I added the version number to the meson build you have to update this line too when you push a new version.

```meson
project('com.github.matfantinel.reminduck', 'vala', 'c', version: '1.2.1')
```

**Comment by NathanBnm** [comment]
I have some issues with French translation. They doesn't appear after the update. No idea is it's related. I'll check on my side.

**Comment by matfantinel** [comment]
Fixed in v1.4.0.
